FRINXio/frinx-inventory-server
# arango collection renamed

HOW TO TEST:

docker-compose up
yarn run arango:seed

you should see renamed collections phy_connected, phy_device, phy_interface, phy_has in localhost:8529
you can test it with frontend if topology is working.
